app/models/move.rb

```diff
@@ -53,7 +55,7 @@ class Move < VersionedModel
   validates :from_location, presence: true
   validates :to_location, presence: true, unless: :prison_recall?
   validates :move_type, inclusion: { in: move_types }
-  validates :profile, presence: true, unless: -> { requested? || cancelled? }
+  validates :profile, presence: true, unless: -> { requested? || booked? || cancelled? }
```

Comment:
Are you sure about this change? i.e. can a move can be booked (confirmed by the supplier) without a person specified?

Reply:
@smoothcontract hmm, that doesn't look correct - I'll fix it